.github/workflows/integration.yml

```diff
@@ -38,6 +39,10 @@ jobs:
             CLIENT_USE_M2CRYPTO: Yes
           - SERVER_USE_M2CRYPTO: No
             CLIENT_USE_M2CRYPTO: No
+          # Fail deprecation
```

Comment:
To be consistent with the other options:
```suggestion
          ###### Fail deprecation
```

Reply:
done